elements/pfe-icon-panel/src/pfe-icon-panel.json

```diff
@@ -0,0 +1,99 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Icon Panel",
+  "description": "This element creates a header, body, and footer region in which to place content or other components.",
+  "type": "object",
+  "tag": "pfe-icon-panel",
+  "class": "pfe-icon-panel",
+  "category": "combo",
+  "properties": {
+    "slots": {
+      "title": "Slots",
+      "description": "Definition of the supported slots",
+      "type": "object",
+      "properties": {
+        "header": {
+          "title": "Header",
+          "type": "array",
+          "namedSlot": true,
+          "items": {
+            "title": "Header item",
+            "oneOf": [
+              {
+                "$ref": "raw"
+              }
+            ]
+          }
+        },
+        "body": {
+          "title": "Body",
+          "type": "array",
+          "namedSlot": false,
+          "items": {
+            "title": "Body item",
+            "oneOf": [
+              {
+                "$ref": "raw"
+              }
+            ]
+          }
+        },
+        "footer": {
+          "title": "Footer",
+          "type": "array",
+          "namedSlot": true,
+          "maxItems": 3,
+          "items": {
+            "title": "Footer item",
+            "oneOf": [
+              {
+                "$ref": "raw"
+              }
+            ]
+          }
+        }
+      }
+    },
+    "attributes": {
+      "title": "Attributes",
+      "type": "object",
+      "properties": {
+        "icon": {
+          "title": "Icon",
+          "type": "string"
+        },
+        "color": {
+          "title": "Background color of circle",
+          "type": "string",
+          "enum": [
+            "lightest",
+            "lighter",
+            "base",
+            "darker",
+            "darkest",
+            "complement",
+            "accent"
+          ],
+          "default": "base"
+        },
+        "centered": {
+          "title": "Centered",
+          "type": "Boolean",
+          "default": false
+        },
+        "stacked": {
+          "title": "Stacked",
+          "type": "Boolean",
+          "default": false
+        },
+        "circled": {
+          "title": "Circled",
+          "type": "Boolean",
+          "default": false
```

Comment:
@starryeyez024 Do you think this should this be true too for the default?

Reply:
Hmm, good question! I'm thinking that usually we opt into flair, so maybe the circle should be an add-on.

Reply:
👍